tokio/src/stream/mod.rs

```diff
@@ -0,0 +1,93 @@
+//! Stream utilities for Tokio.
+//!
+//! `Stream`s are an asynchoronous version of standard library's Iterator.
+//!
+//! This module provides helpers to work with them.
```

Comment:
Can we add something to the docs explaining the relationship between `tokio::stream::Stream` and `futures_core::stream::Stream`? I don't want to confuse users by creating the impression that these types are incompatible, so it would be good to note that the `Stream` trait is a re-export.

Similarly, it would be good to clarify that `StreamExt` is _not_ a re-export. I can see that being confusing.